README.md

```diff
@@ -53,6 +53,14 @@ If the command has a non-zero exit code, a `NonZeroExitCodeException` is thrown
 $"The process exited with code {ExitCode}."
 ```
 
+### Windows
+
+Sometimes, for whatever wonderful reasons, it's necessary to run a different command on Windows. For example, when running [Yarn](https://yarnpkg.com), some combination of mysterious factors may require that you explicitly run `cmd.exe` with Yarn as an argument, rather than running Yarn directly. The optional `windowsNames` and `windowsArgs` parameters may be used to specify a different command name and arguments for Windows:

```

Comment:
```suggestion
Sometimes when running batch scripts, it's necessary to run a different command on Windows. For example, when running [Yarn](https://yarnpkg.com), which has both `yarn.cmd` and `yarn`, it may require that you explicitly run `cmd.exe` with Yarn as an argument, rather than running Yarn directly. The optional `windowsNames` and `windowsArgs` parameters may be used to specify a different command name and arguments for Windows:
```

Reply:
@robertcoltheart thanks for the suggestion but I think I'd prefer to avoid getting into the specifics of _why_ another command on Windows is sometimes necessary. TBH I don't really understand it, and I'm not sure I even want to.